package.json

```diff
@@ -64,14 +66,18 @@
     "debug": "^2.3.3",
     "electron-installer-dmg": "^0.1.2",
     "electron-packager": "^8.4.0",
+    "electron-sudo": "malept/electron-sudo#fix-linux-sudo-detection",
     "electron-winstaller": "^2.5.0",
     "fs-promise": "^1.0.0",
     "github": "^7.2.0",
     "glob": "^7.1.1",
     "inquirer": "^2.0.0",
     "lodash.template": "^4.4.0",
     "log-symbols": "^1.0.2",
+    "node-fetch": "^1.6.3",
```

Comment:
`nugget` is just for the fancy UI, I could remove it but I think the feedback is good for such a big download.